go.mod

```diff
@@ -11,13 +11,9 @@ require (
 	github.com/VictoriaMetrics/fasthttp v1.1.0
 	github.com/VictoriaMetrics/metrics v1.22.2
 	github.com/VictoriaMetrics/metricsql v0.45.0
-	github.com/aws/aws-sdk-go v1.44.100
+	github.com/aws/aws-sdk-go v1.44.101
```

Comment:
why this was affected by the change?

Reply:
i just run `make vendor-update` that is it